lib/rubygems/source.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 autoload :FileUtils, 'fileutils'
 
+require "uri"
```

Comment:
Can you move to this in the `initialize` method?

Reply:
updated